configure.ac

```diff
@@ -671,6 +682,10 @@ AC_TRY_COMPILE([],[
 		[Define to 1 if you need to use #pragma pack instead of __attribute__ ((packed))])]
 )
 
+xdefault_intf_is_enabled="xenable_intf_${DEFAULT_INTF}"
+if test "x${!xdefault_intf_is_enabled}" != "xyes"; then
+	AC_MSG_ERROR([** Cannot set ${DEFAULT_INTF} as default; ${DEFAULT_INTF} is not enabled. :${!xdefault_intf_is_enabled}:])
```

Comment:
Apparently, this fails the build for MacOS where the default 'open' interface isn't enabled. I honestly don't know what is the default interface for Mac or why 'open' isn't enabled there (probably, some libs are missing). Nonetheless without your changes it was building fine somehow. This build failure must be fixed.

Reply:
It is funny that the part that I added to make it work more intuitively broke things. I added this part to be able to print out the default value of the default interface variable. And on Linux, that is open. I added a check to see if open is not available (or not building) and default to lan, which is likely universally built in that case. I tested it by forcing a build that disables open.